rest/routes/lastVisited.go

```diff
@@ -25,18 +26,20 @@ func StoreLastVisitedPages(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = service.HandlePostLastVisitedPages(recentPages.Pages, userId)
+	err = service.HandlePostLastVisitedPages(recentPages.Pages, user)
 	if err != nil {
 		panic(err)
 	}
-	pages, err := service.GetUsersLastVisitedPages(userId)
+
+	fmt.Println("&&&&&&&&&&&&&&", user.LastVisitedPages.Data())
```

Comment:
You can keep it, just move it to a debug logger :)

Reply:
:rofl: no I was just trying to figure out what the .Data does